Jenkinsfile

```
@@ -13,6 +13,9 @@ def isDevelopBranch = env.BRANCH_NAME == developBranchName
 def daysToKeep = isDevelopBranch ? '7' : '-1'
 def numToKeep = isDevelopBranch ? '-1' : '10'
 
+def shortTimeout = 10
+def longTimeout = 45
```

```suggestion
def shortTimeoutMinutes = 10
def longTimeoutMinutes = 45
```

Or just a comment above saying the unit is minutes, otherwise I can already predict questions :slightly_smiling_face: